CONTRIBUTING.md

```diff
@@ -20,3 +20,19 @@ If you added a contributor by mistake, you can remove them in a comment with:
 
 If you are making a pull request on behalf of someone else but you had no part in designing the 
 feature, you can remove yourself with the above syntax.
+
+# Tests
+
+For normative changes for any specification in
+[CR or later](https://www.w3.org/Style/CSS/current-work) as well as the pre-CR specifications listed
+below, a corresponding [web-platform-tests](https://github.com/w3c/web-platform-tests) PR must be
```

Comment:
I've been asking for a policy like this, so obvious this makes me happy, but I think it's important that there's some leeway here. The "testing is not practical" case will probably come up at least occasionally at CR or later, and if everyone agrees, then this "must" ought not get in the way of progress. I think that if this is downgraded to a "should", then editor's can use their best judgement about when to file issues instead of requiring tests.

Reply:
The way I'm reading it, the 'must' is to have a wpt PR *or* an explanation/issue for followup. It's still at the editor's discretion, but they must do one or the other.

Reply:
Perhaps I'm reading this too much like a format spec, but I'm not able to parse it that way, "a corresponding PR must be provided" sounds quite final. Of course, it's only with RFC 2119 goggles that changing it to "should" helps. Interested to hear how others read this, and don't mind it if other have a more lenient interpretation.

Reply:
I think if the intent is the way @astearns is reading it, then it needs to be reworded to be more clear about the either-or.